README.md

```diff
@@ -1,73 +1,23 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
 # Project Name
```

Comment:
-  [ ] Please kindly replace this placeholder text (`Project Name`) with the title of your project(`for example "Portfolio Page" `), as this will make your `readme file` more informative and professional.